pwndbg/gdblib/regs.py

```diff
@@ -48,21 +37,15 @@ class module(ModuleType):
     def __getattr__(self, attr: str) -> int:
         attr = attr.lstrip("$")
         try:
-            # Seriously, gdb? Only accepts uint32.
-            if "eflags" in attr or "cpsr" in attr:
-                value = gdb77_get_register(attr)
```

Comment:
We removed this part: did we check if `pwndbg.gdblib.regs.eflags` work after this?

Reply:
(also random thought: `x in attr` was so wrong :D)